### PR TITLE
Add configurable title to Lovelace thermostat card

### PR DESCRIPTION
**Pull request by girlpunk**

### CODE REVIEW

File: src/panels/lovelace/cards/hui-thermostat-card.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ const modeIcons = {
 
 interface Config extends LovelaceConfig {
   entity: string;
+  title: string | null;
```
Comment (zsarnett) [inline src/panels/lovelace/cards/hui-thermostat-card.ts]: No need for the `| null` just use `title?: string;`

File: src/panels/lovelace/cards/hui-thermostat-card.ts [hunk 2]
```diff
@@ -81,7 +83,7 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
         <div id="root">
           <div id="thermostat"></div>
           <div id="tooltip">
-            <div class="title">${computeStateName(stateObj)}</div>
+            <div class="title">${title}</div>
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-thermostat-card.ts]: `this.config.title || computeStateName(stateObj)`

File: src/panels/lovelace/cards/hui-thermostat-card.ts [hunk 3]
```diff
@@ -66,6 +67,7 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
       return html``;
     }
     const stateObj = this.hass.states[this.config.entity] as ClimateEntity;
+    const title = this.config.title == null ? this.config.title : computeStateName(stateObj);
```
Comment (zsarnett) [inline src/panels/lovelace/cards/hui-thermostat-card.ts]: This can be done in line where the title goes as `this.config.title || computeStateName`

Comment (zsarnett) [reply src/panels/lovelace/cards/hui-thermostat-card.ts]: oops didnt realize balloob was reviewing

**Comment by zsarnett** [comment]
Also this PR is against the Master. Needs to be against Dev

**Pull request closed without merging**